assets/app/styles/_tile.less

```diff
@@ -5,11 +5,13 @@
 .tile {
   background: @console-panel-color;
   .panel_default_boxshadow();
-  padding: 10px 20px; 
+  padding: 10px 20px;
   margin-bottom: 20px;
   word-wrap: break-word;
```

Comment:
`word-break: break-all` does break short words when we don't want.

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/8357713/4a1ac8e2-1b2a-11e5-9c09-1bd5642ed88c.png)

I'm not sure why `break-word` isn't doing what we want. It is set.

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/8357739/7018413c-1b2a-11e5-9ecf-ff22bb0e5a9e.png)